internal/app/user/user.go

```diff
@@ -0,0 +1,35 @@
+package user
```

Comment:
Please help to create a folder `services `in api folder, then move this package into it